content-webapp/src/main/webapp/news/components/NewsApp.vue

```diff
@@ -202,28 +215,32 @@ export default {
     }
   },
   created() {
-    const filterQueryParam = this.getQueryParam('filter');
-    const searchQueryParam = this.getQueryParam('search');
-    const spacesFilterParam = this.getQueryParam('spaces')?.split('_');    
-    if (filterQueryParam || searchQueryParam || spacesFilterParam) {
-      if (filterQueryParam) {
-        // set filter value, which will trigger news fetching
-        this.newsFilter = filterQueryParam;
-      }
-      if (searchQueryParam) {
-        // set search value
-        this.searchText = searchQueryParam;
-      }
-      if (spacesFilterParam) {
-        // set search value
-        this.spacesFilter = spacesFilterParam;
+    this.$featureService.isFeatureEnabled('NewsScheduleAndFilterDisplaying').then(enabled => {
+      this.NewsScheduleAndFilterDisplaying = enabled;
```

Comment:
```suggestion
this.$featureService.isFeatureEnabled('NewsScheduleAndFilterDisplaying').then(enabled => this.NewsScheduleAndFilterDisplaying = enabled);
```

Reply:
This should be executed inside the "then" method after initializing the "NewsScheduleAndFilterDisplaying"  to correctly compute the filter options !